Automattic/wp-calypso
# TailoredFlows: remove support for old newsletter intro design

#### Proposed Changes

* With the [PR to update the newsletter intro design](https://github.com/Automattic/wp-calypso/pull/69388) we gave support for 2 different newsletter designs to exist while the translations for the PR were in progress.
* Since the translations [are now complete](https://translate.wordpress.com/deliverables/overview/7595429) we can remove the support for the old newsletter intro design and only show the new one

#### Testing Instructions
- Checkout this branch or use the PR image link
- Check the newsletter intro step and see if it matches the new designs
- Check the newsletter intro in a locale different than `en`
- Check the link in bio step for possible regressions

Related #69387
